packages/crawler/tests/e2e/helpers/puppeteer-helper.ts

```diff
@@ -0,0 +1,121 @@
+import type { Page, WaitForOptions } from 'puppeteer';
+import { getLoaderScript } from '@porsche-design-system/components-js/partials';
+
+export type PdsTestingContext = {
+  bodyHtml: string;
+  firstPdsVersionPrefixes?: string[];
+  secondPdsVersionPrefixes?: string[];
+};
+export const getInternalLoaderScriptForPrefixes = (prefixes: string[]): string =>
+  prefixes.reduce((result, prefix) => result + getLoaderScript({ prefix: prefix }), '');
+
+export const getExternalLoaderScriptForPrefixes = (prefixes: string[]): string => {
+  const prefixesLoaders = prefixes.reduce(
```

Comment:
Could be replaced by `.map().join()` for less multiline string interpolation.

Reply:
Fixed